sdks/python/test-suites/gradle.properties

```diff
@@ -27,3 +27,6 @@ dataflow_chicago_taxi_example_task_py_versions=3.7
 
 # direct test-suites
 direct_mongodbio_it_task_py_versions=3.5
+
+# portable test-suites
+portable_flink_validates_runner_py_versions=3.6
```

Comment:
Can we add all supported Python versions?

Reply:
done